corehq/apps/es/migrations/0009_no_op_migration.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.23 on 2024-05-09 12:54
+# This migration exists as a placeholder migration which was merged but rolled back before the deploy.
+# If any third party hoster has deployed during that time period,
+# it ensures that next migration that we add shows up for them.
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('es', '0008_add_doc_id_to_all_mappings'),
+    ]
+
+    operations = [
+        migrations.RunPython(migrations.RunPython.noop, reverse_code=migrations.RunPython.noop)
```

Comment:
Why doesn't this do anything

Reply:
Chatted with Cal and Daniel about this and ended up creating https://github.com/dimagi/commcare-hq/pull/34591 so that you could potentially review/approve this when we are offline. This PR includes a more thorough solution to cleanup these indices if a self hoster did end up running the reverted migration.